external/os-guides/centos/centos-minimal.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: doc
+title: CentOS Template
```

Comment:
This should be `CentOS Minimal Template` to distinguish it from the full `CentOS Template`.